tests/test_dle.py

```diff
@@ -413,49 +399,6 @@ def test_loops_ompized(fa, fb, fc, fd, t0, t1, t2, t3, exprs, expected, iters):
                 assert 'omp for' not in k.value
 
 
-@skipif_yask
-def test_loop_nofission(simple_function):
-    old = Rewriter.thresholds['min_fission'], Rewriter.thresholds['max_fission']
-    Rewriter.thresholds['max_fission'], Rewriter.thresholds['min_fission'] = 0, 1
-    handle = transform(simple_function, mode='fission')
-    assert """\
-  for (int i = 0; i <= 3; i += 1)
-  {
-    for (int j = 0; j <= 5; j += 1)
-    {
-      for (int k = 0; k <= 7; k += 1)
-      {
-        a[i] = a[i] + b[i] + 5.0F;
-        a[i] = -a[i]*c[i][j] + b[i]*d[i][j][k];
-      }
-    }
-  }""" in str(handle.nodes)
-    Rewriter.thresholds['min_fission'], Rewriter.thresholds['max_fission'] = old
-
-
-@skipif_yask
-def test_loop_fission(simple_function_fissionable):
-    old = Rewriter.thresholds['min_fission'], Rewriter.thresholds['max_fission']
-    Rewriter.thresholds['max_fission'], Rewriter.thresholds['min_fission'] = 0, 1
-    handle = transform(simple_function_fissionable, mode='fission')
-    assert """\
- for (int i = 0; i <= 3; i += 1)
-  {
-    for (int j = 0; j <= 5; j += 1)
-    {
-      for (int k = 0; k <= 7; k += 1)
-      {
-        a[i] = a[i] + b[i] + 5.0F;
-      }
-      for (int k = 0; k <= 7; k += 1)
-      {
-        b[i] = a[i] + pow(b[i], 2) + 3;
-      }
-    }
-  }""" in str(handle.nodes)
-    Rewriter.thresholds['min_fission'], Rewriter.thresholds['max_fission'] = old
-
-
```

Comment:
Any reason for removing tests?

Reply:
In this PR, I've dropped obsolete functionalities, among which a DLE pass called ``loop fission``. Here, I'm just dropping its tests